packages/rocketchat-livechat/livechat.js

```diff
@@ -29,17 +29,30 @@ WebApp.connectHandlers.use('/livechat', Meteor.bindEnvironment((req, res, next)
 
 	const head = Assets.getText('public/head.html');
 
+	let baseUrl;
+	if (__meteor_runtime_config__.ROOT_URL_PATH_PREFIX && __meteor_runtime_config__.ROOT_URL_PATH_PREFIX.trim() !== '') {
+		baseUrl = __meteor_runtime_config__.ROOT_URL_PATH_PREFIX;
+	} else {
+		baseUrl = '/';
+	}
+	if (/\/$/.test(baseUrl) === false) {
+		baseUrl += '/';
+	}
+
```

Comment:
A variable (**baseUrl**) was created to deal with **__meteor_runtime_config __. ROOT_URL_PATH_PREFIX**, it would not be nice to use this same variable below, where **__meteor_runtime_config __. ROOT_URL_PATH_PREFIX** is mentioned again?